### PR TITLE
Add import of goa UUID during generation of user_type

### PR DESCRIPTION
**Pull request by aslakknutsen**

### CODE REVIEW

File: goagen/gen_client/generator_test.go [hunk 1]
```diff
@@ -222,4 +222,57 @@ var _ = Describe("Generate", func() {
...
+			codegen.TempCount = 0
+			testType := &design.UserTypeDefinition{
+				AttributeDefinition: &design.AttributeDefinition{
+					Type: design.Object{
+						"param": &design.AttributeDefinition{Type: design.Integer},
+						"time":  &design.AttributeDefinition{Type: design.DateTime},
+						"uuid":  &design.AttributeDefinition{Type: design.UUID},
+					},
+				},
+				TypeName: "TesType",
```
Comment (kwk) [inline goagen/gen_client/generator_test.go]: Should be `TestType`, right?

Comment (aslakknutsen) [inline goagen/gen_client/generator_test.go]: Yeah, doesn't matter for the test itself, but consistency FTW! :)

**Comment by raphael** [comment]
Thank you! Do you think you could also send a PR against the `v1` branch?

**Comment by aslakknutsen** [comment]
@raphael of course, https://github.com/goadesign/goa/pull/841 :)

**Pull request merged**